pkg/server/singleprocess/auth.go

```diff
@@ -133,20 +133,26 @@ func (s *service) cookieFromRequest(ctx context.Context) string {
 	return ""
 }
 
+func (s *Service) AuthenticateCookie(
+	ctx context.Context, token, endpoint string, effects []string,
+) {
+
+}
```

Comment:
This doesn't appear to be used anywhere, perhaps left over?

Reply:
Good catch, thank you!